packages/ida.diaphora.vm/ida.diaphora.vm.nuspec

```diff
@@ -2,11 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ida.diaphora.vm</id>
-    <version>2.1.0</version>
+    <version>2.1.0.20230617</version>
```

Comment:
@mr-tz we are in the future! 👀  today is 2023-06-16! Well, as long as we don't update this package again, it doesn't matter. 😆